gensim/utils.py

```diff
@@ -381,6 +383,35 @@ def call_on_class_only(*args, **kwargs):
     raise AttributeError('This method should be called on a class object.')
 
 
+class DictWrapper():
```

Comment:
missing docstrings

Reply:
'Wrapper' is not very descriptive of its role/functionality. Seems it should be 'Counter' – and maybe even just using the Python built-in `Counter`?

Reply:
@gojomo 
I'm sorry, I don't understand what you mean by
> Seems it should be 'Counter' – and maybe even just using the Python built-in Counter

especially about the "Python built-in Counter"
Please could you clarify that?

Would a name like `DictCounter` and `BounterCounter` be more suitable?

Reply:
https://docs.python.org/2/library/collections.html#collections.Counter